src/styles/cylc/_tree.scss

```diff
@@ -64,10 +70,10 @@ $icon-size: 1.2em;
     display: block;
 
     .node-expand-collapse-button {
-      width: 1.5em;
-      height: 1.5em;
-      margin: 0 2px;
+      width: $icon-width;
+      height: $icon-width;
       cursor: pointer;
+      user-select: none;
```

Comment:
I noticed rapidly clicking the expand/collapse icon would cause the task name text to become selected. This prevents that

Reply:
We should probably apply this to a lot of other things in the UI.